releasenotes/notes/44605.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: telemetry
+releaseNotes:
+  - |
+    **Added** metric expiry support, when `ISTIO_META_METRIC_ROTATION_INTERVAL` and
```

Comment:
Do we have some reference documentation for these variables?

(putting do-not-merge for others to complete the review)

Reply:
IMO, this worth a doc in istio.io?